pages.zh/common/2to3.md

```diff
@@ -0,0 +1,32 @@
+# 2to3
+
+> 自动将 Python 2 代码转换成 Python 3。
+> 更多信息：<https://docs.python.org/3/library/2to3.html>.
+
+- 显示将执行的变动但不执行（dry-run）：
```

Comment:
I think these colons used are incorrect. I think it should be `:` rather than `：`

Reply:
Per our rules it should, but since the parentheses before it is full-width I'm not entirely sure. @blueskyson?

Reply:
Current way complies the Chinese-specific rules and looks good to me as well. See also the last example in [hexo](https://github.com/tldr-pages/tldr/blob/main/pages.zh/common/hexo.md) use the same way.